scripts/test/hyriseBenchmarkTPCH_test.py

```diff
@@ -48,7 +48,7 @@ def main():
     benchmark.expect_exact("Using prepared statements: yes")
     benchmark.expect_exact("Loading/Generating tables done")
     benchmark.expect_exact("Sorting tables")
-    benchmark.expect_exact("Creating index on customer [ c_custkey ]")
+    benchmark.expect_exact("No indexes defined by benchmark")
```

Comment:
How is that related to `arguments["--chunk_indexes"] = "true"`?

Reply:
It is because the `--chunk_indexes` flag tells the BenchmarkRunner to create indexes, but since we decided to remove them it can't find any to create and prints this message.

Reply:
I thought they are removed by the next PR following this one here?

Reply:
Indeed.
The TableGenerator outputs `No indexes defined by benchmark` when he it is instructed to generate indexes, but none are defined, regardless of index type.

Reply:
Ah okay. Right, you removed the indexes from the benchmark. That's another issue I had. :D